From eae795f478861c71b1d75d3805f2ca874e2cff18 Mon Sep 17 00:00:00 2001
From: Steffen Jaeckel <jaeckel-floss@eyet-services.de>
Date: Wed, 30 Mar 2022 16:40:06 +0200
Subject: [PATCH] fix installation of manpages
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

fixup of 46b9428d92e86612fafc198bb3575acdf6167649

[0] writes something about "... use ‘man*section*_MANS’ over ‘man_MANS’ ..."
in the context of "fast installation", so I tried that and it worked.

TBH I don't really understand why this fixes the problem.

[0] https://www.gnu.org/software/automake/manual/html_node/Man-Pages.html

Signed-off-by: Steffen Jaeckel <jaeckel-floss@eyet-services.de>
---
 Makefile.am | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index cfeb4f7d7..0e35a518f 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -221,7 +221,7 @@ icons_sources = $(top_srcdir)/icons/*
 
 script_sources = bootstrap.sh
 
-man_sources = $(wildcard docs/profanity*.1)
+man1_sources = $(wildcard $(top_srcdir)/docs/profanity*.1)
 
 if BUILD_PGP
 core_sources += $(pgp_sources)
@@ -289,9 +289,9 @@ tests_unittests_unittests_LDADD = -lcmocka
 #endif
 #endif
 
-man_MANS = $(man_sources)
+man1_MANS = $(man1_sources)
 
-EXTRA_DIST = $(man_sources) $(icons_sources) $(themes_sources) $(script_sources) profrc.example theme_template LICENSE.txt README.md CHANGELOG
+EXTRA_DIST = $(man1_sources) $(icons_sources) $(themes_sources) $(script_sources) profrc.example theme_template LICENSE.txt README.md CHANGELOG
 
 # Ship API documentation with `make dist`
 EXTRA_DIST += \
